docs/UserGuide.md

```diff
@@ -345,22 +321,104 @@ Examples:
 * `c-find John` returns `john` and `John Doe`.
 * `c-find alex david` returns `Alex Yeoh`, `David Li`.<br>
 
-#### 3.7 Locating persons based on matching tags: `c-tag-find`
+#### 3.5 Locating employees based on matching tags: `c-tag-find`
 
-Finds all active(unarchived) contacts that contain the KEYWORD(s) in their tags.
+Finds all employees whose tags contain the one or more `KEYWORD(s)`.
 
 Format: `c-tag-find KEYWORD [MORE_KEYWORDS] ...`
 
-* The search is case-insensitive. e.g `Friday` will match `friday`.
-* The order of the keywords does not matter. e.g. `friday monday` will match `monday friday`.
-* Only the tag names are searched.
-* Only full words will be matched e.g. `PartTime` will not match `PartTimes`.
-* Employees whose tag(s) matching at least one keyword will be listed in the `Employee Contact DeatailL` pane
- (i.e. `OR` search).  e.g. `Friday Monday` will return employees whose tags contain `Friday` or `Monday`.
+* The search is case-insensitive. e.g `Friday` will match `friday`.<br>
+* The order of the keywords does not matter. e.g. `friday monday` will match `monday friday`.<br>
+* Only the tag names are searched.<br>
+* Only full words will be matched. e.g. `Fri` will not match `Friday`.<br>
+* Employees whose tag(s) matching at least one keyword will be listed in the _Employee Directory_ pane
+ (i.e. `OR` search).  e.g. `Friday Monday` will find employees whose tags contain `Friday` or `Monday`.<br>
+
 
 Examples:
-* `c-tag-find friday` returns all employees whose tags contain `friday`.
-* `c-tag-find friday parttime` returns all employees whose tags contain `friday`, `parttime`.<br>
+* Let's say you want to find out who are the available to work on `Monday` or `Friday`. In this case, the two key
+ words are `Monday` and `Friday`. You can follow these steps.
+
+Steps to find employees who are available to work on `Monday` or `Friday`:
+1. Type `c-tag-find monday friday` into the _Command Box_.
+2. Press `Enter` to execute.
+
+Outcome:
+1. The _Result Display_ will show the total number of employees who are can work on `Friday` or `Monday`.
+
+2. Employees who are can work on `Friday` or `Monday` will be listed in the _Employee Directory_ pane.
+
+{Screenshot}
+
+#### 3.6 Locating all employees who are available to work today: `c-today`
+
+Finds employees whose tags contain today's day (i.e. `Wednesday`, `Tuesday`, etc).
+
+Format: `c-today`
+
+* The search is case-insensitive (e.g `Friday` will match `friday`).<br>
+* The application will automatically get today's day based on the current computer's date.<br>
```

Comment:
I think that "computer's current date" is a better phrasing.

Reply:
Thanks!